src/test/suite/rectangle-mark-mode.test.ts

```diff
@@ -70,6 +70,37 @@ KLMNOPQRST`;
     expect(activeTextEditor.selections).toEqual([new vscode.Selection(1, 2, 1, 2)]);
   });
 
+  test("killing and yanking in rectangle-mark-mode", async () => {
+    setEmptyCursors(activeTextEditor, [1, 2]);
+    const emulator = new EmacsEmulator(activeTextEditor);
+
+    emulator.rectangleMarkMode();
+
+    await emulator.runCommand("forwardChar");
+    await emulator.runCommand("forwardChar");
+    await emulator.runCommand("nextLine");
+    await emulator.runCommand("nextLine");
+
+    expect(activeTextEditor.selections).toEqual([
+      new vscode.Selection(1, 2, 1, 4),
+      new vscode.Selection(2, 2, 2, 4),
+      new vscode.Selection(3, 2, 3, 4),
+    ]);
+
+    await emulator.runCommand("killRegion");
+
+    assertTextEqual(
+      activeTextEditor,
+      `0123456789
+abefghij
+ABEFGHIJ
+klopqrst
+KLMNOPQRST`
+    );
+
+    // expect(activeTextEditor.selections).toEqual([new vscode.Selection(3, 2, 3, 2)]); // TODO: Now not supported
```

Comment:
There is still a problem, but it's skipped in this version with experimental rectangle-mark-mode support